config/v1/types_infrastructure.go

```diff
@@ -531,9 +531,296 @@ type OvirtPlatformStatus struct {
+	// virtual machine is created/located.
+	// +kubebuilder:validation:Optional
+	// +kubebuilder:validation:MinLength=1
+	// +kubebuilder:validation:MaxLength=2048
+	Folder string `json:"folder,omitempty"`
+}
+
+// VSpherePlatformDeploymentSuitable
+// +kubebuilder:validation:Enum=Allowed;NotAllowed
+type VSpherePlatformDeploymentSuitable string
+
+const (
+	// Allowed indicates that the Deployment Zone is suitable for
+	// control plane nodes.
+	Allowed VSpherePlatformDeploymentSuitable = "Allowed"
+
+	// NotAllowed indicates that the Deployment Zone is not suitable for
+	// control plane nodes.
+	NotAllowed VSpherePlatformDeploymentSuitable = "NotAllowed"
+)
+
+// VSpherePlatformDeploymentZone holds the association between a
+// vCenter, failure domain and the virtual machine placementConstraints
+type VSpherePlatformDeploymentZone struct {
+	// name defines the VSpherePlatformDeploymentZoneSpec name.
+	// +kubebuilder:validation:MinLength=1
+	// +kubebuilder:validation:MaxLength=256
+	// +kubebuilder:validation:Required
+	Name string `json:"name"`
+
+	// server is the fully-qualified domain name or the IP address of the vCenter server.
+	// +kubebuilder:validation:Required
+	// +kubebuilder:validation:MinLength=1
+	// +kubebuilder:validation:MaxLength=255
+	Server string `json:"server"`
+
+	// failureDomain is the name of the VSphereFailureDomain used for this VSphereDeploymentZone
+	// +kubebuilder:validation:Required
+	// +kubebuilder:validation:MinLength=1
+	// +kubebuilder:validation:MaxLength=256
+	FailureDomain string `json:"failureDomain"`
+
+	// ControlPlane determines if this failure domain is suitable for use by control plane machines.
+	// There is three valid options: unset, Allowed and NotAllowed.
+	// +kubebuilder:validation:Optional
+	ControlPlane VSpherePlatformDeploymentSuitable `json:"controlPlane,omitempty"`
+
+	// PlacementConstraint encapsulates the placement constraints
+	// used within this deployment zone.
+	// +kubebuilder:validation:Optional
+	PlacementConstraint VSpherePlatformPlacementConstraint `json:"placementConstraint"`
+}
+
+// VSpherePlatformNodeNetworkingSpec holds the network CIDR(s) and port group name for
+// including and excluding IP ranges in the cloud provider.
+// This would be used for example when multiple network adapters are attached to
+// a guest to help determine which IP address the cloud config manager should use
+// for the external and internal node networking.
+type VSpherePlatformNodeNetworkingSpec struct {
+	// networkSubnetCidr IP address on VirtualMachine's network interfaces included in the fields' CIDRs
+	// that will be used in respective status.addresses fields.
+	// +kubebuilder:validation:Optional
+	NetworkSubnetCIDR []string `json:"networkSubnetCidr,omitempty"`
+
+	// network VirtualMachine's VM Network names that will be used to when searching
+	// for status.addresses fields. Note that if InternalNetworkSubnetCIDR and
```

Comment:
Whats InternalNetworkSubnetCIDR? Did we rename it?

Reply:
Thanks for the catch. We changed from upstream capv the api design.
Fixing